### PR TITLE
Fixes a bunch of helper scripts for improved usability

### PR DESCRIPTION
**Pull request by mmattel**
This PR fixes many necessary helper scripts which eases maintenance and different repo setups.

* find orphand adoc files
-- fix missed hits,
single multi repo capability (one script for all repos!)
* find orphand attachments/example/images
-- create a uniformed master script which is called by a task script defining the environment,
-- single and multi repo capability (one script for all repos!)
* rename the `manual_config` script to `manual_config_pdf`
-- ease of differenciation
* deleted two image filens which were orphaned (there are many more, but for testing this was ok)

Backport to 10.9 and 10.8 (must be done immediately post merging)

### CODE REVIEW

File: bin/find-orphaned-attachment-files [hunk 1]
```diff
@@ -1,35 +1,27 @@
 #!/usr/bin/env bash
 
 #
-# This script checks the three core ownCloud documentation manuals to 
-# see if any of the attachment files are orphaned.
+# This script checks the ownCloud documentation manual(s) to 
+# see if attachment/example/image files are orphaned.
```
Comment (phil-davis) [inline bin/find-orphaned-attachment-files:5]:
```suggestion
# see if attachment files are orphaned.
```

File: bin/find-orphaned-attachment-files [hunk 2]
```diff
@@ -1,35 +1,27 @@
...
+BASE_DIR=""
+MANUAL_POSTFIX="_manual"
+TASK="ATTACHMENT"
+TARGET_DIR="assets/attachments"
+# depending on the target, there may be a trailing slash necessary or not
+# check how the file is included
+SEARCH_PATTERN='link:{attachmentsdir}/'
 
-list=$(echo ${AVAILABLE_MANUALS[@]} | sed -r 's/[ ]+/, /g')
+# get the full path the the directory the script runs
```
Comment (phil-davis) [inline bin/find-orphaned-attachment-files:20]:
```suggestion
# get the full path of the directory where the script runs
```

File: bin/find-orphaned-base [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+		#echo "${BASE_DIR}"
+		#echo "${BASE_DIR}/${TARGET_DIR}"
+		#echo "${BASE_DIR}/pages"
+		#exit
+
+		[[ ! -e "${BASE_DIR}" ]] && echo "Cannot find ${manual} manual." && continue
+
+		echo -e "\e[1;32mChecking the ${manual} manual \e[0m"
+
+		# return all files found, but omit an error message if TARGET_DIR is not present
```
Comment (phil-davis) [inline bin/find-orphaned-base:57]:
```suggestion
		# return all files found, but emit an error message if TARGET_DIR is not present 
```

File: bin/find-orphaned-example-files [hunk 4]
```diff
@@ -1,35 +1,27 @@
 #!/usr/bin/env bash
 
 #
-# This script checks the three core ownCloud documentation manuals to 
-# see if any of the example files are orphaned.
+# This script checks the ownCloud documentation manual(s) to 
+# see if attachment/example/image files are orphaned.
```
Comment (phil-davis) [inline bin/find-orphaned-example-files:5]:
```suggestion
# see if example files are orphaned.
```

File: bin/find-orphaned-image-files [hunk 5]
```diff
@@ -1,35 +1,27 @@
 #!/usr/bin/env bash
 
 #
-# This script checks the three core ownCloud documentation manuals to 
-# see if any of the image files are orphaned.
+# This script checks the ownCloud documentation manual(s) to 
+# see if attachment/example/image files are orphaned.
```
Comment (phil-davis) [inline bin/find-orphaned-image-files:5]:
```suggestion
# see if image files are orphaned.
```

File: bin/find-orphaned-image-files [hunk 6]
```diff
@@ -1,35 +1,27 @@
...
+BASE_DIR=""
+MANUAL_POSTFIX="_manual"
+TASK="IMAGE"
+TARGET_DIR="assets/images"
+# depending on the target, there may be a trailing slash necessary or not
+# check how the file is included
+SEARCH_PATTERN='image:'
 
-list=$(echo ${AVAILABLE_MANUALS[@]} | sed -r 's/[ ]+/, /g')
+# get the full path the the directory the script runs
```
Comment (phil-davis) [inline bin/find-orphaned-image-files:20]:
```suggestion
# get the full path to the directory where the script runs
```

File: bin/find-orphaned-example-files [hunk 7]
```diff
@@ -1,35 +1,27 @@
...
+BASE_DIR=""
+MANUAL_POSTFIX="_manual"
+TASK="EXAMPLE"
+TARGET_DIR="examples"
+# depending on the target, there may be a trailing slash necessary or not
+# check how the file is included
+SEARCH_PATTERN='include::{examplesdir}'
 
-list=$(echo ${AVAILABLE_MANUALS[@]} | sed -r 's/[ ]+/, /g')
+# get the full path the the directory the script runs
```
Comment (phil-davis) [inline bin/find-orphaned-example-files:20]:
```suggestion
# get the full path to the directory where the script runs
```

File: bin/find-orphaned-image-files [hunk 8]
```diff
@@ -1,35 +1,27 @@
...
+DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 
-echo
-echo "Checking for orphaned IMAGE files in manual: ${list}."
-echo "Please run this script from your docs root"
-echo
+# load the workinng script
```
Comment (phil-davis) [inline bin/find-orphaned-image-files:23]:
```suggestion
# load the working script
```

File: bin/find-orphaned-attachment-files [hunk 9]
```diff
@@ -1,35 +1,27 @@
...
+DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 
-echo
-echo "Checking for orphaned ATTACHMENT files in manual: ${list}."
-echo "Please run this script from your docs root"
-echo
+# load the workinng script
```
Comment (phil-davis) [inline bin/find-orphaned-attachment-files:23]:
```suggestion
# load the working script
```

File: bin/find-orphaned-example-files [hunk 10]
```diff
@@ -1,35 +1,27 @@
...
+DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 
-echo
-echo "Checking for orphaned EXAMPLE files in manual: ${list}."
-echo "Please run this scrip from your docs root"
-echo
+# load the workinng script
```
Comment (phil-davis) [inline bin/find-orphaned-example-files:23]:
```suggestion
# load the working script
```

**Review by phil-davis — APPROVED** [review]

File: bin/find-orphaned-base (hunk 3 above)
Comment (mmattel) [reply bin/find-orphaned-base:57]: omit is correct, do not return an error message if target_dir is not found

**Pull request merged**